doc/introduction/index.rst

```diff
@@ -442,7 +437,16 @@ The Boot process proceeds as follows:
 #. The virtual bootloader starts the User-side verified boot process.
 
 In this boot mode, the boot options are defined via the ``VM{x}_CONFIG_OS_BOOTARGS``
-macro in the source code (replace ``{x}`` with the VM number).
+macro in the source code (replace ``{x}`` with the VM number) by
```

Comment:
Is `VM{x}_CONFIG_OS_BOOTARGS` really still in use? I don't find it anywhere in the code and the only place it's referenced is in the documentation (this file :-))